### PR TITLE
added tests to check users created correctly, fixed user assertions

### PR DESCRIPTION
**Pull request by sbryzak**
Fixes https://issues.redhat.com/browse/CRT-1791

Related member-operator PR: https://github.com/codeready-toolchain/member-operator/pull/433

### CODE REVIEW

**Review by alexeykazakov — COMMENTED** [review]
Looks good! But let's also make sure that all identities are deleted. I can find only one place where we do it: https://github.com/codeready-toolchain/toolchain-e2e/blob/master/test/e2e/e2e_test.go#L470
And it doesn't cover the cases with multiple identities.
Maybe we have something else for that?

**Review by alexeykazakov — APPROVED** [review]
Looks good! But let's add detailed descriptions to the tests.

File: test/e2e/usersignup_test.go [hunk 1]
```diff
@@ -229,6 +229,69 @@ func (s *userSignupIntegrationTest) TestUserIDAndAccountIDClaimsPropagated() {
...
+	identityID := uuid.Must(uuid.NewV4())
+
+	// when
+	userSignup, _ := NewSignupRequest(s.Awaitilities).
+		Username("test-user-with-userid-and-originalsub").
+		Email("test-user-with-userid-and-originalsub@redhat.com").
+		IdentityID(identityID).
+		UserID(identityID.String()).
+		AccountID("abc-8783").
+		OriginalSub("def:98734987234").
```
[the remark this answers is not in the document]
Comment (sbryzak) [reply test/e2e/usersignup_test.go:305]: Added comments to new tests

**Comment by mfrancisc** [comment]
/retest

done rebase of member-operator

**Pull request merged**